dasbus/server/interface.py

```diff
@@ -298,8 +298,8 @@ def _generate_interface(cls, interface_cls, interfaces, interface_name):
                 element = cls._generate_method(member, member_name)
             else:
                 raise DBusSpecificationError(
-                    "{}.{} cannot be exported.".format(
-                        interface_cls.__name__, member_name
+                    "Unsupported definition of DBus member '{}'.".format(
+                        member_name
```

Comment:
In some places (such as here) the message changes to a radically different meaning. I think it's because you exchange the causes and consequences. Have you considered that both could be useful?

Reply:
The error message is not the only information you usually have. The stack trace can provide the necessary context and some of the messages didn't make sense it this context or just didn't tell anything new. Also, sometimes it is not possible to fully describe the causes and consequences of the error, because the code doesn't have access to all data. It is difficult to decide what could be useful. This PR is based on my current experience with dasbus. The further improvements of the error messages will have to be handled case-by-case.